global_planner:
  type: "Interpolation"

local_planner:
  type: "Interpolation"
  step_size: 0.01 # in Euclidean space
  steps: 10

# IK has been moved to the move_to_pos.yaml file
# inverse_kinematics:
#   type: "relaxed-IK"
#   service: "relaxed_ik/solve_pose"

debugger:
  evaluate_time: True
